build.gradle

```diff
@@ -3,22 +3,27 @@
 buildscript {
 
     ext {
-        kotlin_version = '1.3.71'
-        kotlin_coroutines_version = '1.4.2'
-        jna_version = '5.6.0'
-        android_gradle_plugin_version = '4.0.1'
-        android_components_version = '75.0.0'
+        kotlin_version = '1.5.20'
+        kotlin_coroutines_version = '1.5.0'
+        jna_version = '5.8.0'
```

Comment:
By the way, this version of JNA seems to fix the struct-padding bug from https://github.com/mozilla/uniffi-rs/issues/334 \o/